Automattic/woocommerce-payments
# TotalPaymentActivity component scaffolding as per figma design

Fixes #8516 

#### Changes proposed in this Pull Request
Created `<TotalPaymentActivity />` component scaffolding as per figma design


![image](https://github.com/Automattic/woocommerce-payments/assets/15019298/3a3a638d-d88f-4158-81d7-96a050e3757b)










#### Testing instructions





- Cross-check with Figma design
- Make sure tests don't break

-------------------


**Post merge**
